### PR TITLE
Added example usage of elixir sdk 

### LINKED ISSUE #89: Add example usage of elixir sdk

### PR DESCRIPTION
**Pull request by DanielaIvanova**
resolves #89

### CODE REVIEW

File: examples/usage.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+``` elixir
+public_key = "ak_6A2vcm1Sz6aqJezkLCssUXcyZTX7X8D5UwbuS2fRJr9KkYpRU"
+secret_key = "a7a695f999b1872acb13d5b63a830a8ee060ba688a478a08c6e65dfad8a01cd70bb4ed7927f97b51e1bcb5e1340d12335b2a2b12c8bc5221d63c4bcb39d41e61"
+network_id = "ae_uat"
+url = "https://sdk-testnet.aepps.com/v2"
+internal_url = "https://sdk-testnet.aepps.com/v2"
+client = Core.Client.new(%{public: public_key, secret: secret_key}, network_id, url, internal_url)
+```
+
+And now, you are ready to use **Aepp SDK Elixir**.
```
Comment (d-velev) [inline examples/usage.md]: This line is a bit unnecessary

File: README.md [hunk 2]
```diff
@@ -42,3 +42,5 @@ iex> init_args = ["42"]
 iex> Core.Contract.deploy(client, source_code, init_args)
 {:ok, "ct_2sZ43ScybbzKkd4iFMuLJw7uQib1dpUB8VDi9pLkALV5BpXXNR"}
 ```
+## How to use it
```
Comment (d-velev) [inline README.md]:
```suggestion
## Usage
```

File: README.md [hunk 3]
```diff
@@ -42,3 +42,5 @@ iex> init_args = ["42"]
...
+If you are wondering how you can use in your elixir project, check [here](https://github.com/aeternity/aepp-sdk-elixir/tree/master/examples/usage.md).
```
Comment (d-velev) [inline README.md]:
```suggestion
An installation and usage guide can be found [here](https://github.com/aeternity/aepp-sdk-elixir/tree/master/examples/usage.md).
```

**Comment by d-velev** [comment]
@DanielaIvanova could you also remove the example usage from the readme?

**Comment by DanielaIvanova** [comment]
@d-velev Could you mark which line you would like me to remove?

**Pull request merged**